tests/models/test_rope_scaling.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2024 MosaicML LLM Foundry authors
+# SPDX-License-Identifier: Apache-2.0
+
+from transformers.models.llama.modeling_llama import LlamaRotaryEmbedding
```

Comment:
can you add a test that checks that the new llama3 embedding type from HF outputs the same embeddings when `apply_scaling` is called as the rope modification code I shared with you?

Reply:
i don't think that is necessary. we don't need to test _everything_ from hf